nuitka/freezer/Onefile.py

```diff
@@ -170,13 +170,19 @@ def packDistFolderToOnefileLinux(onefile_output_filename, dist_dir, binary_filen
     stdout_file = openTextFile(stdout_filename, "wb")
     stderr_file = openTextFile(stderr_filename, "wb")
 
+    if options.appimage_compression == "gzip":
```

Comment:
One of the benefits of having a choice, is that you get to not check here anymore, you can just use it here, not even with a variable at all.

I think `subprocess` and friends would hate having one string with a space or an empty value (which you don't need anymore), they would be passed in `argv` as a value, and not e.g. be ignoring for empty string, or become two entries, so actually AppImage ought to error out with this code, but if it doesn't, it surely is not generally possible.

So  `"--comp,getAppImageCompression(),` would be good there I suppose.

Reply:
I've implemented this change, could you please confirm I haven't missed anything? thanks a bunch!
And you're right, appimagetool was giving me a weird error with that code (strange because it worked the first time then never again)